pkg/reconcilermanager/controllers/build_names.go

```diff
@@ -21,19 +21,27 @@ import (
 	"kpt.dev/configsync/pkg/core"
 )
 
+const (
+	// RepoSyncBaseClusterRoleName is the namespace reconciler permissions name.
+	// e.g. configsync.gke.io:ns-reconciler
+	RepoSyncBaseClusterRoleName = configsync.GroupName + ":" + core.NsReconcilerPrefix
+	// RootSyncBaseClusterRoleName is the root reconciler base ClusterRole name.
+	// e.g. configsync.gke.io:root-reconciler
+	RootSyncBaseClusterRoleName = configsync.GroupName + ":" + core.RootReconcilerPrefix
+	// RepoSyncBaseRoleBindingName is the name of the default RoleBinding created
+	// for RepoSync objects. This contains basic permissions for RepoSync reconcilers
+	//(e.g. RepoSync status update).
+	RepoSyncBaseRoleBindingName = RepoSyncBaseClusterRoleName
+	// RootSyncLegacyClusterRoleBindingName is the name of the legacy ClusterRoleBinding created
+	// for RootSync objects. It is always bound to cluster-admin.
+	RootSyncLegacyClusterRoleBindingName = RootSyncBaseClusterRoleName
+	// RootSyncBaseClusterRoleBindingName is the name of the default ClusterRoleBinding created
+	// for RootSync objects. This contains basic permissions for RootSync reconcilers
+	// (e.g. RootSync status update).
+	RootSyncBaseClusterRoleBindingName = RootSyncBaseClusterRoleName + "-base"
```

Comment:
Is it a breaking change by adding a trailing `-base`?

Reply:
This name is for a new binding, so this is not breaking. The old cluster-admin binding name is unchanged